Modules/EMCAL/src/DigitsQcTask.cxx

```diff
@@ -95,21 +107,32 @@ void DigitsQcTask::monitorData(o2::framework::ProcessingContext& ctx)
     if (!trg.getNumberOfObjects())
       continue;
     QcInfoLogger::GetInstance() << QcInfoLogger::Debug << "Next event " << eventcouter << " has " << trg.getNumberOfObjects() << " digits" << QcInfoLogger::endm;
-    gsl::span<const o2::emcal::Digit> eventdigits(digitcontainer.data() + trg.getFirstEntry(), trg.getNumberOfObjects());
+    //gsl::span<const o2::emcal::Digit> eventdigits(digitcontainer.data() + trg.getFirstEntry(), trg.getNumberOfObjects());
+    gsl::span<const o2::emcal::Cell> eventdigits(digitcontainer.data() + trg.getFirstEntry(), trg.getNumberOfObjects());
     for (auto digit : eventdigits) {
       int index = digit.getHighGain() ? 0 : (digit.getLowGain() ? 1 : -1);
       if (index < 0)
         continue;
 
       mDigitAmplitude[index]->Fill(digit.getEnergy(), digit.getTower());
       mDigitTime[index]->Fill(digit.getTimeStamp(), digit.getTower());
-      //if we fill phy vs eta plots integrated: filled with eta phi GlobalRowColumnFromIndex  from Geometry
 
       // get the supermodule for filling EMCAL/DCAL spectra
+
       try {
+        // fill here mDigitOccupancy for column and rows
+        //if we fill phy vs eta plots integrated: filled with eta phi GlobalRowColFromIndex  from Geometry
+
+        auto [row, col] = mGeometry->GlobalRowColFromIndex(digit.getTower());
+        if (digit.getEnergy() > 0)
+          mDigitOccupancy->Fill(col, row);
+        mIntegratedOccupancy->Fill(col, row, digit.getEnergy());
+        std::cout << " fill histo integrated occupancy  and occupancy with col" << col << " row " << row << " energy as weight " << digit.getEnergy() << std::endl;
```

Comment:
Please use some logging facility instead of `cout` to make sure that the messages reach infologger in production.